test/libsolidity/syntaxTests/viewPureChecker/eof/inline_assembly_instructions_disallowed_view.sol

```diff
@@ -0,0 +1,30 @@
+contract C {
+    function f() public view {
+        assembly {
+            sstore(0, 1)
+            pop(extcall(0, 1, 2, 3))
+            pop(extdelegatecall(0, 1, 2))
+            log0(0, 1)
+            log1(0, 1, 2)
+            log2(0, 1, 2, 3)
+            log3(0, 1, 2, 3, 4)
+            log4(0, 1, 2, 3, 4, 5)
+
+            // These two are disallowed too but the error suppresses other errors.
+            //pop(msize())
+            //pop(pc())
+        }
+    }
+}
+// ====
+// EVMVersion: >=london
+// bytecodeFormat: >=EOFv1
```

Comment:
I see that some cases `viewPureChecker/eof/inline_assembly_instructions_*.sol` have EVM version restriction. It's unnecessary.

Reply:
Done